addon-test-support/@ember/test-helpers/dom/-get-element.js

```diff
@@ -0,0 +1,16 @@
+import { getContext } from '../setup-context';
+
+export default function getElement(selectorOrElement) {
+  if (
+    selectorOrElement instanceof Window ||
+    selectorOrElement instanceof Document ||
+    selectorOrElement instanceof HTMLElement ||
+    selectorOrElement instanceof SVGElement
+  ) {
+    return selectorOrElement;
+  }
+
+  let rootElement = getContext().element;
+
+  return rootElement.querySelector(selectorOrElement);
```

Comment:
should we typecheck if `selectorOrElement` is a string?

Reply:
I thought about that, we _can_ but I was unsure if we could do beat things by allowing Object#toString to be leveraged...

I do not feel strongly though...